sig-docs/charter.md

```diff
@@ -0,0 +1,97 @@
+# SIG Docs Charter
+
+This charter adheres to the conventions described in the [Kubernetes Charter README] and adheres to the Roles and Organization Management specified in [sig-governance].
+
+## Scope
+
+SIG Docs publishes Kubernetes documentation on kubernetes.io. Kubernetes documentation includes:
+  - Documentation of the core Kubernetes APIs
+  - Core Kubernetes architecture 
+  - CLI tools shipped with the Kubernetes release
+
+ Responsibility for creating feature documentation belongs to the developers and SIGs associated with a feature. This includes documentation for features, concepts, components, opinions, processes, etc.
+
+SIG Docs sets standards for feature documentation, provides clear paths for docs contribution, and [coordinates documentation updates][docs-release] during quarterly releases.
+
+SIG Docs maintains the Kubernetes website's infrastructure and tooling.
+
+SIG Docs creates subprojects as needed to handle specific aspects of Kubernetes documentation.
+
+### In scope
+
+SIG [readme]
+
+#### Code, Binaries and Services
+
+- The kubernetes.io website:
+
+  - Infrastructure and tooling
+  - Support and standards for internationalization (i18n) and localization (L10n)
+  - Maintaining and applying the style guide to feature documentation and release notes
+  - Reviewing and publishing release notes during quarterly release cycles
+  - Generated reference documentation for:
+    - The core Kubernetes APIs
+    - Kubernetes command-line tools, including but not limited to kubectl
+    - Kubernetes setup tools
+    - The Federation API
+
+See Cross-cutting for limitations on how the SIG helps with related efforts on other projects.
+
+#### Cross-cutting and Externally Facing Processes
+
+- [Standards for content][standards]: style guide, contributor guide, PR reviews
```

Comment:
I think we should call out the doc contribution guide separately.

Reply:
I left this sentence in place but added a hyperlink to the previous "code, binaries, and services" section.